src/main/java/core/basesyntax/exception/RegistrationIllegalArgumentException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class RegistrationIllegalArgumentException extends IllegalArgumentException {
```

Comment:
I recommend using raw RuntimeException
```suggestion
public class RegistrationIllegalArgumentException extends RuntimeException {
```

Reply:
amended